packages/medusa/src/loaders/module-definitions/index.ts

```diff
@@ -0,0 +1,34 @@
+import glob from "glob"
+import path from "path"
+import resolveCwd from "resolve-cwd"
+
+import { ModuleDefinition, ModuleResolution } from "../../types/global"
+
+export default (configModule) => {
+  const moduleResolutions = {} as Record<string, ModuleResolution>
+  const projectModules = (configModule.modules as Record<string, string>) ?? {}
+
+  const definitionsDir = path.join(__dirname, "*.{j,t}s")
+  const definitions = glob.sync(definitionsDir, {
+    ignore: ["**/index.js", "**/index.ts", "**/*.d.ts"],
+  })
```

Comment:
self-review: Follow same patterns as with feature flags. Module definitions will be added in `module-definitions/`

Reply:
We can expect module definitions to be more static than feature flags (e.g., a feature flag will typically be removed, modules will likely not). For this reason, I suggest we have one large file with all the definitions in an object. This object could of course later be composed of imports from smaller files when/if it gets too big. 

However, I don't see the need for dynamically imported definitions based on globs.

Reply:
See 18a70dc